src/task/style.css

```diff
@@ -258,3 +258,24 @@ footer {
   color: rgb(255, 92, 92);
   font-size: 0.75rem;
 }
+
+#assigneeAvatar {
+  background-position: right;
+  background-size: 42px;
+  position: absolute;
+  top: 33%;
+  width: 40px;
+  height: 40px;
+  right: 3%;
+  border-radius: 50%;
+  display: none;
+}
+
+#assigneeAvatarLoader {
+  position: absolute;
+  top: 41%;
+  right: 3%;
+  display: none;
+  border: 3px solid #f3f3f3;
+  border-top: 3px solid #1d58d8;
```

Comment:
Nit: Since the border width and border style is the same for the top border from the line above, you could just set the top border color using[ border-top-color](https://developer.mozilla.org/en-US/docs/Web/CSS/border-top-color) CSS property